### PR TITLE
Remove all AWX_ISOLATED* fields

### PR DESCRIPTION
**Pull request by jakemcdermott**
I believe this should address https://github.com/ansible/awx/issues/10123. The issue mentions a `worker-json.js` 404 and, while we should fix that at some point, I don't think it's actually related to the `save` not working.

##### SUMMARY
I noticed the job settings form wasn't saving on devel. 

After some debugging, I found that removing the `AWX_ISOLATED_CHECK_INTERVAL` and `AWX_ISOLATED_LAUNCH_TIMEOUT` made the form saveable again.

It doesn't seem like we use _any_ of the `AWX_ISOLATED*` fields anymore (not to be confused with `AWX_ISOLATION*` fields, which we do use) so this PR removes them all. 

cc @wenottingham @shanemcd @marshmalien  @gamuniz @tiagodread @jbradberry

### CODE REVIEW

**Comment by jbradberry** [comment]
The settings I removed in #9957 are:

  - AWX_ISOLATED_PUBLIC_KEY
  - AWX_ISOLATED_PRIVATE_KEY
  - AWX_ISOLATED_KEY_GENERATION
  - AWX_ISOLATED_HOST_KEY_CHECKING
  - AWX_ISOLATED_USERNAME
  - AWX_ISOLATED_CONNECTION_TIMEOUT
  - AWX_ISOLATED_LAUNCH_TIMEOUT
  - AWX_ISOLATED_PERIODIC_CHECK
  - AWX_ISOLATED_CHECK_INTERVAL

File: awx/ui_next/src/screens/Host/data.hostFacts.json [hunk 1]
```diff
@@ -112,7 +112,6 @@
       "ANSIBLE_FORCE_COLOR": "True",
       "ANSIBLE_CACHE_PLUGIN": "jsonfile",
       "AWX_PRIVATE_DATA_DIR": "/tmp/awx_13_r1ffeqze",
-      "AWX_ISOLATED_DATA_DIR": "/tmp/awx_13_r1ffeqze/artifacts/13",
```
Comment (jbradberry) [inline awx/ui_next/src/screens/Host/data.hostFacts.json:115]: This one should not be removed.

Comment (jbradberry) [reply awx/ui_next/src/screens/Host/data.hostFacts.json:115]: Never mind, something else must have removed this.

**Review by wenottingham — APPROVED** [review]

**Review by tiagodread — APPROVED** [review]
Verified that the user can save the form successfully.

**Pull request merged**